lib/net.js

```diff
@@ -133,6 +134,7 @@ function Socket(options) {
   this._handle = null;
   this._parent = null;
   this._host = null;
+  this._timeout = null;
```

Comment:
Isn't that part of the implementation of a Timeout instance? i.e. didn't a socket already have it, it just wasn't declared in the constructor?